e2e/explorer/index.spec.ts

```diff
@@ -407,3 +409,51 @@ test(`it is possible to rename a directory`, async ({ page }) => {
   const isRenamed = await checkIfNodeExists(namespace, newname);
   await expect(isRenamed).toBeTruthy();
 });
+
+test(`it is possible to deactivate and activate a workflow`, async ({
+  page,
+}) => {
+  const workflow = "testworkflow";
+  test.setTimeout(50000);
+  await createWorkflow(namespace, `${workflow}.yaml`);
+  await page.goto(`/${namespace}/explorer/workflow/active/${workflow}.yaml`, {
+    waitUntil: "networkidle",
+  });
+  const btnToggle = page.getByTestId("toggle-workflow-active-btn");
+
+  const iconPowerOn = page.getByTestId("active-workflow-on-icon");
+  const iconPowerOff = page.getByTestId("active-workflow-off-icon");
+  const btnRun = page.getByTestId("workflow-header-btn-run");
+
+  await expect(
+    iconPowerOff,
+    "initial state should be active, shows the power-off button"
+  ).toBeVisible();
+  await expect(
+    btnRun,
+    "initial state should be active, run button is active"
+  ).toBeEnabled();
+
+  await btnToggle.click();
+  await actionWaitForSuccessToast(page);
+  await expect(
+    iconPowerOn,
+    "next state should be inactive, shows the power-on button"
+  ).toBeVisible();
+  await expect(
+    btnRun,
+    "next state should be inactive, run button is inactive"
+  ).toBeDisabled();
+
+  await btnRun.click({ force: true }); //nothing happens on this click
+  await expect(btnRun, "run button is still inactive").toBeDisabled();
```

Comment:
If possible, it would be good to check the "run-workflow-dialog" isn't opened here.

This is a bit tricky to implement because you need to make sure it waits for a bit to avoid a false test result (if you check immediately after clicking, the modal will probably always be invisibe even if it is opening).

So to make sure your assertion works, you have to put it in another place where you know that the modal will open, and make sure that it will fail.

Reply:
I have no way to know that nothing happens on page after clicking the run button which is disabled
I think we can simply wait for 1 sec and check the dialog is opened

Reply:
This was implemented with waiting for 1 sec.